uaclient/tests/test_config.py

```diff
@@ -1402,6 +1408,20 @@ def test_process_config(
         assert expected_out.strip() == out.strip()
         assert "" == err
 
+    def test_process_config_errors_for_wrong_timers(self):
+        cfg = UAConfig(
+            {
+                "ua_config": {
+                    "update_messaging_timer": "wrong",
+                    "update_status_timer": 43200,
+                    "gcp_auto_attach_timer": 1800,
+                }
+            }
+        )
+
+        with pytest.raises(exceptions.UserFacingError):
+            cfg.process_config()
```

Comment:
Can we check the exception message here to make sure we failing due to the expected exception ?